### PR TITLE
ITRocket endpoints for mocha

### PR DESCRIPTION
**Pull request by itrocket-team**
## Summary by CodeRabbit

- **New Features**
  - Introduced new endpoints for Celestia testnet, including consensus, RPC, API, and gRPC services.
  - Added a new explorer endpoint for easier access to testnet data.

### CODE REVIEW

File: nodes/mocha-testnet.md [hunk 1]
```diff
@@ -81,6 +81,7 @@ full blocks from it.
 - `full.consensus.mocha-4.celestia-mocha.com`
 - `consensus-full-mocha-4.celestia-mocha.com`
 - `rpc-mocha.pops.one`
+- `celestia-testnet-consensus.itrocket.net`
```
Comment (jcstein) [inline nodes/mocha-testnet.md:84]: what ports are open?

File: nodes/mocha-testnet.md [hunk 2]
```diff
@@ -103,6 +104,7 @@ Celestia network. The default port is 26657.
 - `celestia-t-rpc.noders.services/`
 - `rpc-1.testnet.celestia.nodes.guru`
 - `rpc-2.testnet.celestia.nodes.guru`
+- `celestia-testnet-rpc.itrocket.net`
```
Comment (jcstein) [inline nodes/mocha-testnet.md:107]: what ports are open?

Comment (itrocket-team) [reply nodes/mocha-testnet.md:107]: RPC: 26657
gRPC: 9090

File: nodes/mocha-testnet.md (hunk 1 above)
Comment (itrocket-team) [reply nodes/mocha-testnet.md:84]: RPC: 26657
gRPC: 9090

File: nodes/mocha-testnet.md (hunk 2 above)
Comment (jcstein) [inline nodes/mocha-testnet.md:107]:
```suggestion
- `celestia-testnet-rpc.itrocket.net:443`
```

File: nodes/mocha-testnet.md (hunk 1 above)
Comment (jcstein) [inline nodes/mocha-testnet.md:84]:
```suggestion
- `celestia-testnet-consensus.itrocket.net`
  - RPC port: 26657
  - gRPC port: 9090
```

**Review by jcstein — APPROVED** [review]

**Pull request merged**